### PR TITLE
docs(Accessibility): improve introduction section

### PR DESCRIPTION
**Pull request by jurokapsiar**
This PR improves introduction section to accessibility, mainly summarizing what needs to be done in preparation for correct component usage to achieve accessible experiences

image

### CODE REVIEW

File: docs/src/views/Accessibility.tsx [hunk 1]
```diff
@@ -56,31 +56,70 @@ export default () => (
...
-      {link('ARIA best practises', 'https://www.w3.org/TR/wai-aria-practices-1.1/')}
-      and validates them with subject matter experts.
+      correct ARIA roles, testing on multiple screen reader / os combinations. This allows spending
+      more time on the usability aspects of accessibility.
+    </p>
+    <p>
+      Following steps help to design an accessible user experience:
+      <ol>
+        <li>
+          decompose UI to parts and <b>identify components, variants and behaviors to use</b>
```
Comment (kolaps33) [inline docs/src/views/Accessibility.tsx:78]: do we want to mention "behaviors to use" ?

I mean if we have already default behaviors. What do you think about something like this?: 
"Decompose UI to parts and <b>identify components, variants. Default behavior will apply for component. "

File: docs/src/views/Accessibility.tsx [hunk 2]
```diff
@@ -56,31 +56,70 @@ export default () => (
...
+            {link('color and contrast', 'https://accessibility.umn.edu/core-skills/color-contrast')}
+          </b>{' '}
+          to convey information
+        </li>
+        <li>
+          define <b>tab order and arrow key navigation</b>
+        </li>
+        <li>
+          specify <b>labels</b>, especially for components without textual information (icon only
+          buttons)
```
Comment (kolaps33) [inline docs/src/views/Accessibility.tsx]: Maybe I would add if you agree something like.

"Label the menu, list, toolbars component, to give user more context. "

File: docs/src/views/Accessibility.tsx [hunk 3]
```diff
@@ -56,31 +56,70 @@ export default () => (
...
+        <li>
+          identify UI parts that appear on <b>hover or focus</b> and specify keyboard and screen
+          reader interaction with them
+        </li>
+        <li>
+          list cases when <b>focus</b> needs to be <b>moved programatically</b> (if parts of the UI
+          are appearing/disappearing or other cases){' '}
+        </li>
+        <li>
+          list cases when <b>focus</b> needs to be <b>trapped</b> in sections of the UI
```
Comment (kolaps33) [inline docs/src/views/Accessibility.tsx]: if we want to add note, that focus is usually trapped in dialog(s) ?

File: docs/src/views/Accessibility.tsx (hunk 1 above)
Comment (jurokapsiar) [reply docs/src/views/Accessibility.tsx:78]: for example for menu or popup there are multiple behaviors - they will need to choose which one to use

File: CHANGELOG.md [hunk 4]
```diff
@@ -16,6 +16,10 @@
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Documentation
+- Accessibility: improve introduction section @jurokapsiar ([#1368](https://github.com/stardust-ui/react/pull/1368))
```
Comment (miroslavstastny) [inline CHANGELOG.md:30]: Please follow the order of changelog sectionsas described in INSTRUCTIONS above.

**Review by miroslavstastny — APPROVED** [review]

**Review by sophieH29 — APPROVED** [review]

**Pull request merged**